sql/core/src/test/scala/org/apache/spark/sql/StringFunctionsSuite.scala

```diff
@@ -352,6 +352,44 @@ class StringFunctionsSuite extends QueryTest with SharedSparkSession {
     // scalastyle:on
   }
 
+  test("UTF-8 string is valid") {
+    // scalastyle:off
+    checkAnswer(Seq("大千世界").toDF("a").selectExpr("is_valid_utf8(a)"), Row(true))
```

Comment:
Could you call the new added functions from Scala/Java API instead of the registered functions, please. I would understand that eventually we invoke the SQL function, but the implementation might be changed in the future as it happened already. Please, invoke new functions directly.